### ISSUE TITLE
Connection to latex server times out from time to time

### ISSUE THREAD

**Issue by kasperosterbye** [pillar-markup/Microdown#379]
And it can make tests fail
```
[1184] MicRichTextComposerTest
[1185](https://github.com/kasperosterbye/Microdown/runs/6072966308?check_suite_focus=true#step:4:1185)
 ✗ #testMathEnvironmentError (251ms)
[1186](https://github.com/kasperosterbye/Microdown/runs/6072966308?check_suite_focus=true#step:4:1186)
ConnectionTimedOut: Cannot connect to 45.63.71.69:443
[1187](https://github.com/kasperosterbye/Microdown/runs/6072966308?check_suite_focus=true#step:4:1187)
[ConnectionTimedOut signal: 'Cannot connect to '
[1188](https://github.com/kasperosterbye/Microdown/runs/6072966308?check_suite_focus=true#step:4:1188)
					, (NetNameResolver stringFromAddress: hostAddress) , ':' , port asString] in Socket>>connectTo:port:waitForConnectionFor:
[1189](https://github.com/kasperosterbye/Microdown/runs/6072966308?check_suite_focus=true#step:4:1189)
Socket>>waitForConnectionFor:ifTimedOut:
```

Two things need to be done I think:
1. make a test that will not fail (cache some data for the test)
2. Give better feedback in the case it is not a test

**Comment by kasperosterbye** [comment]
Its fix will be dependent on the resolver - PR #374. So wait with this one.

**Comment by kasperosterbye** [comment]
Fixed
